server/service/src/sync/translations/item.rs

```diff
@@ -101,6 +120,60 @@ impl SyncTranslation for ItemTranslation {
             sync_record.record_id.clone(),
         )))
     }
+
+    fn change_log_type(&self) -> Option<ChangelogTableName> {
+        Some(ChangelogTableName::Item)
+    }
+
+    fn try_translate_to_upsert_sync_record(
+        &self,
+        connection: &StorageConnection,
+        changelog: &ChangelogRow,
+    ) -> Result<PushTranslateResult, anyhow::Error> {
+        let Some(item) = ItemRowRepository::new(connection).find_one_by_id(&changelog.record_id)?
```

Comment:
Tempted to say we should put an `is_central_server` check in here, but I do know there's an idea of draft items coming that Craig's mentioned, so maybe should just mark anything as draft that's from remote?